test/e2e-encryption-rotation/e2e-encryption-rotation_test.go

```
@@ -25,21 +26,25 @@ var provider = flag.String("provider", "aescbc", "encryption provider used by th
 // then it forces a key rotation by setting the "encyrption.Reason" in the operator's configuration file
 func TestEncryptionRotation(t *testing.T) {
 	ctx := context.TODO()
-	cs := operatorencryption.GetClients(t)
+	cs := library.GetClients(t)
 
 	ns := fmt.Sprintf("test-encryption-on-off-%s", rand.String(4))
-	_, err := cs.KubeClient.CoreV1().Namespaces().Create(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ns}}, metav1.CreateOptions{})
+	_, err := cs.Kube.CoreV1().Namespaces().Create(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ns}}, metav1.CreateOptions{})
+	require.NoError(t, err)
+	defer cs.Kube.CoreV1().Namespaces().Delete(ctx, ns, metav1.DeleteOptions{})
```

golangci-lint (2.12.2)

[error] Error return value of `(k8s.io/client-go/kubernetes/typed/core/v1.NamespaceInterface).Delete` is not checked

(errcheck)